deploy/docker-compose-weave/docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       - external
       - secure
       - internal
+      - backoffice
```

Comment:
The front-end pings shipping and queue-master for health, both are only accessible on the backoffice network.